etc/docker/elyra/Dockerfile

```
@@ -48,5 +48,11 @@ RUN conda remove --force -y terminado && \
         jupyter lab build; \
     fi
 
+# Comment out to hide the component examples for Kubeflow Pipelines and Apache Airflow
+RUN if [ "$TAG" != "dev" ] ; then \
+        elyra-metadata install component-registries --schema_name=elyra-airflow-examples-catalog --display_name="Apache Airflow examples" --runtime_type=APACHE_AIRFLOW --categories="['examples']" --version=1 && \
```

TODO after https://github.com/elyra-ai/elyra/pull/2282 and https://github.com/elyra-ai/elyra/pull/2263 were merged

```suggestion
        elyra-metadata install component-catalogs --schema_name=elyra-airflow-examples-catalog --display_name="Apache Airflow examples" --runtime_type=APACHE_AIRFLOW --categories="['examples']" && \
```